.nvmrc

```diff
@@ -1 +1 @@
-23
+22
```

Comment:
```suggestion
24
```

Reply:
for dev + ci (test; lint; type check;) I'm in favor of using LTS

Reply:
LTS is suppose to be stabler than current

Reply:
True, but we don't need stability for this. It's not production, it's a controlled environment. It also helps me as author of many of those "unstable" features test them in the real world and find (and fix problems).

Reply:
We use oxlint with is in beta so don't care about being on "stable" thing let's be fun

Reply:
W00t!